dffml/service/dev.py

```diff
@@ -1056,6 +1061,118 @@ class Bump(CMD):
     packages = BumpPackages
 
 
+class SphinxBuildError(Exception):
+    """
+    Raised when sphinx-build command exits with a non-zero error code.
+    """
+
+    def __str__(self):
+        return "[ERROR] Failed run sphinx, is it installed (pip install -U .[dev])?"
+
+
+@configdataclass
+class MakeDocsConfig:
+    port: int = field("PORT for the local docs server", default=8080)
+    http: str = field(
+        "If set  a SimpleHTTP server would be started to show generated docs.",
+        default=None,
+    )
+
+
+class MakeDocs(CMD):
+    """
+    Generate docs of the complete package 
+    """
+
+    CONFIG = MakeDocsConfig
+
+    async def _exec_cmd(self, cmd: List[str]) -> int:
+        print(f"$ {' '.join(cmd)}")
+        proc = await asyncio.create_subprocess_exec(*cmd)
+        await proc.wait()
+        return proc.returncode
+
+    async def run(self):
+        root = Path(__file__).parents[2]
+        pages_path = root / "pages"
+        shutil.rmtree(pages_path, ignore_errors=True)
+
+        docs_path = root / "docs"
+        files_to_check = [
+            (("changelog.md",), ("CHANGELOG.md",)),
+            (("shouldi.md",), ("examples", "shouldi", "README.md",)),
+            (("swportal.rst",), ("examples", "swportal", "README.rst",)),
+            (
+                ("contributing", "consoletest.md",),
+                ("dffml", "util", "testing", "consoletest", "README.md",),
+            ),
+        ]
+
+        for symlink, source in files_to_check:
+            file_path = docs_path.joinpath(*symlink)
+            if not file_path.exists():
+                file_path.symlink_to(root.joinpath(*source))
+
+        # HTTP Service Docs
+        service_path = docs_path / "plugins" / "service"
+        service_path.mkdir(parents=True, exist_ok=True)
+        http_docs_pth = service_path / "http"
+
+        if http_docs_pth.exists():
+            http_docs_pth.unlink()
+
+        http_docs_pth.symlink_to(root / "service" / "http" / "docs")
+
+        # Main Docs
+        scripts_path = root / "scripts"
+        scripts_to_run = ["docs.py", "docs_api.py"]
+        for script in scripts_to_run:
+            cmd = [sys.executable, str(scripts_path / script)]
+            returncode = await self._exec_cmd(cmd)
+            if returncode != 0:
+                raise RuntimeError
+
+        cmd = [
+            f"{[e.name  for e in pkg_resources.iter_entry_points('console_scripts') if e.name.startswith('sphinx-build')][0]}",
```

Comment:
```suggestion
            [e.name  for e in pkg_resources.iter_entry_points('console_scripts') if e.name.startswith('sphinx-build')][0],
```

Reply:
yes it is a string , so it was just redundant , anyways , changed this

Reply:
Done